packages/runtime/src/helpers/logger/index.ts

```diff
@@ -0,0 +1,3 @@
+// Winston supports JSON by default but this is using a custom formatter
+// It only gives the message and not the timestamp or log level
+export { logger as default } from './winston'
```

Comment:
I had this logger exposed via an index file as I was trying both `winston` and `pino`. Pino is meant to be faster but by default supports JSON logging. Even with their pretty formatter, it would make our logs a little strange with multiple timestamps.

We can move this to a single logger module that only includes `winston` if needed.